### DISCUSSION TITLE
运行web_demo无法正常显示输出

### DISCUSSION THREAD

**Discussion by yimin-demo** [zai-org/ChatGLM3#599] [Bad Case]
gradio 3.50.2
torch 2.1.1+cu118
显卡RTX4070TI

![image](https://github.com/THUDM/ChatGLM3/assets/144526719/8686643c-6d61-4d28-a032-5ef3d785f280)

![image](https://github.com/THUDM/ChatGLM3/assets/144526719/5f24309d-e0b5-48c9-886e-8f7b47900922)

每次运行都这样，输入一个文本后运行10来秒然后什么也不输出，clear也没用，然后再输新的也没用。有时候进程还卡死，需要手动关进程才能把run的py给退了。

**Comment by yimin-demo** [comment]
没事了，gradio版本不兼容。。。

这gradio的兼容性真的太差了，特别是3和4之间，完全无法做到向下兼容。。。

**Reply by zRzRzRzRzRzRzR** [reply]
一定要按照依赖装
